src/lib/config-capabilities.ts

```
@@ -0,0 +1,53 @@
+  auth?: Record<string, unknown>;
+};
+
+/**
+ * True iff the backend's metadata response carries the field this change
+ * targets. Used to skip unsupported changes before we'd PUT to an endpoint
+ * that may silently drop the body.
+ */
+export function metadataSupports(raw: RawMetadata, change: DiffChange): boolean {
+  if (change.section === 'auth' && change.key === 'allowed_redirect_urls') {
+    return (
+      raw?.auth != null &&
```

ESLint

[error] Expected '!==' and instead saw '!='.

(eqeqeq)